test/e2e/suites/gc_unmanaged/gc_unmanaged_test.go

```diff
@@ -67,7 +67,6 @@ var _ = ginkgo.Context("[unmanaged] [gc]", func() {
 		clusterName := fmt.Sprintf("%s-%s", specName, util.RandomString(6))
 
 		configCluster := defaultConfigCluster(clusterName, namespace.Name)
-		configCluster.KubernetesVersion = e2eCtx.E2EConfig.GetVariable(shared.PreCSIKubernetesVer)
```

Comment:
Use latest kubernetes version to create workload cluster, instead of PreCSIKubernetesVer: v1.22.1. So remove it.